matlab/src/matlab/+arrow/+array/TimestampArray.m

```diff
@@ -70,4 +62,26 @@
             time(indices) = convertTo(dates(indices), "epochtime", TicksPerSecond=ticksPerSecond(units));
         end
     end
+
+    methods(Static)
+        function array = fromMATLAB(data, opts)
+            arguments
+                data
+                opts.TimeUnit(1, 1) arrow.type.TimeUnit = arrow.type.TimeUnit.Microsecond
+                opts.InferNulls(1, 1) logical = true
+                opts.Valid
+            end
+            
+            arrow.internal.validate.type(data, "datetime");
+            arrow.internal.validate.shape(data);
+
+            validElements = arrow.internal.validate.parseValidElements(data, opts);
+            ptime = arrow.array.TimestampArray.convertToEpochTime(data, opts.TimeUnit);
```

Comment:
Maybe we should call this variable `epochTime`?

Reply:
done.